rpcs3/rpcs3.vcxproj.filters

```diff
@@ -599,6 +604,9 @@
     <ClCompile Include="QTGeneratedFiles\Debug - LLVM\moc_register_editor_dialog.cpp">
       <Filter>Generated Files\Debug - LLVM</Filter>
     </ClCompile>
+    <ClCompile Include="rpcs3qt\utilities.cpp">
```

Comment:
I don't see the added file "utilities.cpp" in you commit, is it normal?

Reply:
I don't know how that could still be there, but I'll remove it.